apps/web/src/routes/_content/courses/$courseId/$chapterId.tsx

```diff
@@ -311,7 +311,7 @@ const TimelineBig = ({
                     <img
                       src={OrangePill}
                       className={compose(
-                        'absolute inset-0 bottom-0 left-0 m-auto h-8 w-full',
+                        'absolute inset-0 bottom-0 left-0 m-auto h-8 w-full transform translate-x-2',
```

Comment:
Can you remove that "transform translate-x-2" and instead just change the w-2/3 and w-1/3 of the two divs above with w-1/2 ?
Should look better on all sizes 
![image](https://github.com/user-attachments/assets/4325ab08-a5d2-485e-aa15-d51f3906bb73)
![image](https://github.com/user-attachments/assets/87c4befc-06d0-4d48-9366-ad17b7679c96)
![image](https://github.com/user-attachments/assets/fa01cddc-8568-4eda-ab5a-5b0b1fa90f3f)
![image](https://github.com/user-attachments/assets/1eea5ffa-f897-4e8b-a7a2-566a651ffec3)